src/Reveal.Sdk.Dom.Tests/Visualizations/GridVisualizationFixture.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Visualizations;
+using Reveal.Sdk.Dom.Visualizations.Settings;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations
+{
+
+    public class GridVisualizationTests
+    {
+        [Fact]
+        public void GridVisualization_DefaultConstructor_ShouldInitializeWithDefaultValues()
+        {
+            // Act
+            var gridVisualization = new GridVisualization();
+
+            // Assert
+            Assert.NotNull(gridVisualization);
```

Comment:
Check if we need further Asserts, for ex, Title, dataSourceItem (b/c title, dataSourceItem updates are included in the codes of the constructor)

Reply:
For this, I think I will create other tests to verify which each constructor class to create a object, so for this constructor function, I only verify which it being created by that constructor